public/app/app.ts

```diff
@@ -146,7 +147,9 @@ function initEchoSrv() {
     }
   });
 
-  registerEchoBackend(new PerformanceBackend({}));
+  if (contextSrv.user.orgRole !== '') {
+    registerEchoBackend(new PerformanceBackend({}));
+  }
```

Comment:
Not sure this fixes the issue @torkelo. The backend still gets registered on the login page and the metrics are sent, yet I don't get why it doesn't throw the Unauthorized.. error now?

Reply:
ah, and the get role for myself on the login page is `Viewer`. Is this ok?

Reply:
@dprokop with anonymous access disabled I do not get this backend registered nor any metrics sent

Reply:
@dprokop do you have anonymous access enabled? if not then that is strange

Reply:
you are right, its the anonym access.